cmd/minishift/cmd/addon/util.go

```diff
@@ -41,14 +41,14 @@ func GetAddOnManager() *manager.AddOnManager {
 func WriteAddOnConfig(addOnConfigMap map[string]*addon.AddOnConfig) {
 	c, err := config.ReadConfig()
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Unable to read the Minishift configuration: %s", err.Error()))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Cannot read the Minishift configuration: %s", err.Error()))
```

Comment:
What is the difference between `unable` or `cannot` here. Just trying to understand.